### PR TITLE
Add a path to stdout overflow error

### PR DESCRIPTION
**Pull request by RichardLitt**
See #4

### CODE REVIEW

File: bin/git-unsaved.js [hunk 1]
```diff
@@ -39,7 +39,11 @@ new Tilda(`${__dirname}/../package.json`, {
     gitUnsaved(path, (err, data) => {
         spinner.clear();
         if (err) {
-            console.log(`${colorIt(`${err.message}`).red()}`);
+            if (err.message == 'stdout maxBuffer exceeded') {
+              console.log(`${colorIt(`${err.message} at ${path}`).red()}`);
```
Comment (IonicaBizau) [inline bin/git-unsaved.js]: I think we can use this always, without an `if-else`. `err.message` and the `path` will exist always.

**Comment by RichardLitt** [comment]
Removed unnecessary conditional.

**Review by IonicaBizau — APPROVED** [review]
:rocket:

**Comment by IonicaBizau** [comment]
@RichardLitt I'm [in the mountains now](http://ionicabizau.net/blog/35-the-joy-of-being-a-mentor), but I'm experimenting with an improvised antenna—let's see if I can merge the PR. It's going to be the first pull request merged with my experimental technology. 😁 

Thanks again!

**Comment by RichardLitt** [comment]
@IonicaBizau Fuck yeah! Do you have a description of your mountain set-up, anywhere? Would love to learn what you're using.

**Comment by IonicaBizau** [comment]
I wrote about it on my Patreon profile: https://patreon.com/IonicaBizau <https://patreon.com/IonicaBizau> 😊

Right now, it's very slow, but I pushed the branch. Eventually it will work. The upload speed is terribly slow: 0 Mbps. I have no idea how this message arrives on your side. 😂

**Comment by IonicaBizau** [comment]
This is the first post where I wrote about it: https://www.patreon.com/posts/one-of-my-summer-8502581

Apparently, this worked! :tada:

**Pull request merged**